gobblin-metrics-libs/gobblin-metrics-base/src/main/java/org/apache/gobblin/metrics/event/EventSubmitter.java

```diff
@@ -79,23 +79,53 @@ public EventSubmitter build() {
     }
   }
 
+  public EventSubmitter(MetricContext context) {
```

Comment:
This is the effective the same as `static submit`. I suggest removing it as we're introducing duplicate options and may lead to many undesired usage of `EventSubmitter`.  I'm not opinionated about this one, though.

Reply:
I agree but TimingEvent takes an event submitter as part of the constructor. we will have always use (new EventSubmitter(context, Maps.newHashMap()) when initializing timing events if we dont want to specify additional matadata